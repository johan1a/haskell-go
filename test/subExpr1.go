func main() {
2 - 3  
}
