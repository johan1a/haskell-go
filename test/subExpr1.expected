[SimpleStmt (ExpressionStmt (BinExpr (AritmExpr (SubExpr (Num 2) (Num 3)))))]
